pkg/operator/controller/garden/reconciler_reconcile.go

```diff
@@ -192,6 +203,11 @@ func (r *Reconciler) reconcile(
 			Fn:           flow.Parallel(etcdMain.Wait, etcdEvents.Wait),
 			Dependencies: flow.NewTaskIDs(deployEtcds),
 		})
+		_ = g.Add(flow.Task{
+			Name:         "Deploying Istio",
+			Fn:           r.deployIstioFunc(istio),
+			Dependencies: flow.NewTaskIDs(syncPointSystemComponents),
+		})
```

Comment:
I would rather consider istio a system component, so let's move it up to the first "block".